src_c/time.c

```diff
@@ -278,10 +274,11 @@ time_set_timer(PyObject *self, PyObject *arg)
 
 /*clock object interface*/
 typedef struct {
-    PyObject_HEAD int last_tick;
-    int fps_count, fps_tick;
-    float fps;
-    int timepassed, rawpassed;
+    PyObject_HEAD
+    int juststarted;
+    int fps_count;
+    clock_t last_tick, fps_tick;
+    float fps, timepassed, rawpassed;
     PyObject *rendered;
 } PyClockObject;
```

Comment:
I know it's not part of this PR... but if you feel like cleaning it up:

Our own pygame code should not start with Py, since that's supposed to be for python things.

Reply:
Ok I will replace that with Pg